scripts/build_release.sh

```diff
@@ -95,6 +95,11 @@ make build
 export BUILD_DEB=1
 scripts/build_packages.sh "${PLATFORM}"
 
+# build docker release package
+pwd
+cd ../docker/release
+./build_algod_docker.sh ${HOME}/node_pkg/node_dev_linux-amd64_${FULLVERSION}.tar.gz
```

Comment:
`node_dev_linux` needs to be `node_${CHANNEL}_linux`

Reply:
Added parameter CHANNEL and updated artifact naming to include CHANNEL in addition to FULLVERSION.